### DISCUSSION TITLE
[Answered] Start date

### DISCUSSION THREAD

**Discussion by VladIPlay** [ranahaani/GNews#64] [Q&A]
Hello, for me is not working the start date and end date. 
My code is 
`google_news = GNews(language='ro', country='RO', max_results=100)
google_news.start_date = (2021, 1, 1)
google_news.end_date = (2021, 2, 1)`
but the result are from 2023. 
Even trying with the example code from https://github.com/ranahaani/GNews/blob/master/main.py, i have the same problem.

**Comment by VladIPlay** [comment]
I resolved the problem by downloading directly from github using 
`# use whatever you use to uninstall any pre-existing gnews module
pip uninstall gnews
# install from the project's git main branch
pip install git+https://github.com/ranahaani/GNews.git`

Answer found on https://stackoverflow.com/questions/73072802/web-scraping-articles-from-google-news

**Comment by Charlotte-br560** [comment]
Have you checked if there are any updates or changes to the library that might affect date filtering?
